crates/red_knot/src/main.rs

```diff
@@ -16,7 +17,34 @@ use red_knot_module_resolver::{
     set_module_resolution_settings, RawModuleResolutionSettings, TargetVersion,
 };
 use ruff_db::files::system_path_to_file;
-use ruff_db::system::{OsSystem, System, SystemPath};
+use ruff_db::system::{OsSystem, System, SystemPath, SystemPathBuf};
+
+#[derive(Debug, Parser)]
+#[command(
+    author,
+    name = "red-knot",
+    about = "An experimental multifile analysis backend for Ruff"
+)]
+#[command(version)]
+struct Args {
+    #[clap(help = "File to check", required = true, value_name = "FILE")]
+    entry_point: SystemPathBuf,
+    #[arg(
+        long,
+        value_name = "DIRECTORY",
+        help = "Custom directory to use for stdlib typeshed stubs",
+        env = "RUFF_CUSTOM_TYPESHED_DIR"
```

Comment:
Do we need the environment variable? I would prefer to start with the absolute minimum

Reply:
no, we can leave this for now. though I expect we will want to let users specify it via environment variable as well at some point